ARMTemplates/Archimedes Capacity Scenario/readme.md

```diff
@@ -0,0 +1,214 @@
+# Capacity Scenario template setup
+
+- [Capacity Scenario template setup](#capacity-scenario-template-setup)
+  - [Overview](#overview)
+  - [Installing Pre-reqs](#installing-pre-reqs)
+  - [Synapse pipeline template](#synapse-pipeline-template)
+  - [PBI report template](#pbi-report-template)
+
+
+## Overview
+
+Capacity scenario is a storage use case powered by our available Sharepoint datasets. This allows customers to better understand how their storage is being used by providing meaningful insights and analytics offered by the SharePoint Sites and SharePoint Files datasets
+
+**IMPORTANT NOTE:** This scenario is not currently available publicly. It is coming soon. The details provided here are for informational purposes only.
+
+**After you follow these steps, you will have a great set of Power BI dashboards related to SharePoint Capacity Scenario, like the one shown below.**
+![](Images/Image_1.png) 
+
+## Installing Pre-reqs
+The first step to running this template would be to create an application in the tenant and use that appId 
+and secret to setup the other required resources.
+
+1. Navigate to app registrations in your subscription.
+
+![](Images/1.png)
+
+2. Register a new application
+
+![](Images/2.png)
+
+3. Save the application id (In the screenshot, the one ending in 9826). Navigate to API permissions
+
+![](Images/3.png)
+
+4. Select "Microsoft Graph" from the Add permission flyout
+
+![](Images/4.png)
+
+5. Select "Application permissions -> Applications -> Application.Read.All"
```

Comment:
Is this permissions required? I remember getting some push back on this one.

Reply:
these steps are the same as from the security template. if the instructions have changed there, i can change it here too